### PR TITLE
Web Cmdlets 6.0.0 Documentation Refresh

### PR DESCRIPTION
**Pull request by markekraus**
This PR attempts to refresh the web cmdlet documentation for 6.0.0. It includes all up to date parameters, features, and support

* Reference PowerShell/PowerShell#5492
* Documents PowerShell/PowerShell#5329 `-SslProtocol`
* Restores missing syntax and updates syntax to 6.0.0-rc1
* Removes examples no longer valid in PowerShell 6.0.0
* Updates examples to match 6.0.0 usage
* Removes `FILE` and `FTP` schema mentions as they are no longer supported
* Replaces `Windows PowerShell` with `PowerShell` where appropriate
* Aligns document with formatting standards
* Documents PowerShell/PowerShell#5402 
* Adds security warnings to `-SkipCertificateCheck`
* Remove `-InformationAction` and `-InformationVariable` common paramter from paramters list
* Alphabetized Parameters
* Include `"User@Domain.Com"` username example
* Synced wording of shared parameters between `Invoke-WebRequest` and `Invoke-RestMethod`
* Add platform support notes for `-Certificate`, `-CertificateThumbprint`, and `-SslProtocol`
* Add correct Output types for `Invoke-WebRequest` and `Invoke-RestMethod`
* Documents basic Parsing Only support in 6.0.0

Version(s) of document impacted
------------------------------
- [X] Impacts 6 document
- [ ] Impacts 5.1 document
- [ ] Impacts 5.0 document
- [ ] Impacts 4.0 document
- [ ] Impacts 3.0 document

Reason(s) for not updating all version of documents
--------------------------------------------------
- [X] The documented feature was introduced in version (list version here) of PowerShell
- [ ] This issue only shows up in version (list version(s) here) of the document
- [ ] This PR partially fixes the issue, and issue #<insert here> tracks the remaining work

/cc @joeyaiello

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md](https://github.com/markekraus/PowerShell-Docs/blob/WebCmdletsRefresh/reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md](https://github.com/markekraus/PowerShell-Docs/blob/WebCmdletsRefresh/reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C17%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711172124062040-1870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=REynM2tSWhfmV2NUItMdcA%2BFS%2FC4txV5vFdYcR62fFQ%3D&st=2017-11-17T21%3A47%3A38Z&se=2017-12-18T21%3A52%3A38Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md [hunk 1]
```diff
@@ -57,66 +98,48 @@ PowerShell Security at DerbyCon                                      Wed, 28 Sep
 DSC Resource Kit September Release                                   Thu, 22 Sep 2016 00:25:37 +0000
 PowerShell DSC and implicit remoting broken in KB3176934             Tue, 23 Aug 2016 15:07:50 +0000
 PowerShell on Linux and Open Source!                                 Thu, 18 Aug 2016 15:32:02 +0000
+
+#>
 ```
 
-This command uses the `Invoke-RestMethod` cmdlet to get information from the Windows PowerShell Blog RSS feed.
-The command uses the `Format-Table` cmdlet to display the values of the **Title** and **pubDate** properties of each blog in a table.
+This command uses the `Invoke-RestMethod` cmdlet to get information from the PowerShell Blog RSS feed.
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: According to the Style Guide: 
- When talking about a cmdlet in paragraph, use ` to highlight cmdlet names
- Property names and parameter names should be in bold

Can you fix this in your PR?

Comment (markekraus) [reply reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: hyphen or no hyphen on the parameters, or does it matter?

Comment (sdwheeler) [reply reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: I don't think it matters. I think having the hyphen is useful to distinguish the difference between parameters and properties when talking about both.

Comment (markekraus) [reply reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: agreed. I'll fix them.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md](https://github.com/markekraus/PowerShell-Docs/blob/WebCmdletsRefresh/reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md) | :white_check_mark:Succeeded | |
[reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md](https://github.com/markekraus/PowerShell-Docs/blob/WebCmdletsRefresh/reference/6/Microsoft.PowerShell.Utility/Invoke-WebRequest.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C11%5C18%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201711180103267144-1870%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=7MK07HnSoBw88rHM6aV1qt44DMiaLztgBEg5K%2BQrkBI%3D&st=2017-11-18T01%3A26%3A35Z&se=2017-12-19T01%3A31%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md (hunk 1 above)
Comment (markekraus) [reply reference/6/Microsoft.PowerShell.Utility/Invoke-RestMethod.md]: fixed.

**Pull request merged**